src/base.cls.php

```diff
@@ -373,6 +374,7 @@ class Base extends Root
 
 		// Purge
 		self::O_PURGE_ON_UPGRADE => false,
+		self::O_PURGE_GUEST_PAGES => true,
```

Comment:
Did you notice all the bool vars in this `_default_options` are all `false`? There must be a reason ;)
They are just to define the type, not to set the value.

Reply:
changed the value